contrib/whisper-auto-resize.py

```diff
@@ -119,15 +119,17 @@ def processMetric(fullPath, schemas, agg_schemas):
             archive_config = [archive.getTuple() for archive in schema.archives]
             break
 
-    xFilesFactor = 0.5
-    aggregationMethod = 'average'
-
     # loop through the carbon-aggregation schemas
     for agg_schema in agg_schemas:
-        if agg_schema.matches(metric) and all(x is not None for x in agg_schema.archives):
```

Comment:
Why remove the `None` check here? Won't the `agg_schema.archives` assignment fail below as a result?

```python
>>> a, b = None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not iterable
```